code/game/gamemodes/gang/gang_items.dm

```diff
@@ -284,6 +284,14 @@
 		if(obj.density)
 			to_chat(user, "<span class='warning'>There's not enough room here!</span>")
 			return FALSE
+	
+	var/open = 0
```

Comment:
nothing would, it would be global, taking in the center of the range as an argument.
eg: ```/proc/is_dominator_blocked_at(atom/A)```